src/test/java/com/papertrail/profiler/jaxrs/CpuProfileResourceTest.java

```diff
@@ -1,37 +1,48 @@
 package com.papertrail.profiler.jaxrs;
 
-import org.junit.Test;
+import static junit.framework.TestCase.assertEquals;
 
 import java.io.IOException;
 import java.util.concurrent.CountDownLatch;
 
-import static junit.framework.TestCase.assertEquals;
+import org.junit.Test;
 
-public class CpuProfileResourceTest {
```

Comment:
Please adhere to the code style that was here before, rather than reformatting to change the spacing, indentation, and curly braces placement.